x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/circle.tsx

```diff
@@ -110,14 +104,14 @@ export const Circle: FunctionComponent = () => {
             options: [
               {
                 value: 'shape',
-                label: i18n.translate(
+                text: i18n.translate(
```

Comment:
The `EuiSelect` component expects a `text` prop ([docs](https://elastic.github.io/eui/#/forms/form-controls#select)).

Resolves the warning: ```Warning: Failed prop type: The prop `options[0].text` is marked as required in `EuiSelect`, but its value is `undefined`.```